File: 프로그래머스/0/120839. 가위 바위 보/가위 바위 보.swift
import Foundation

func solution(_ rsp:String) -> String {
    let rspDics: [String:String] = ["2":"0", "0":"5", "5":"2"];
    var result:String = "";
    for ch in rsp {
        result += rspDics[String(ch)]!;
    }
    return result
}

File: 프로그래머스/0/120839. 가위 바위 보/가위 바위 보.js
function solution(rsp) {
    var answer = '';
    
    // 0: 묵, 2: 찌, 5: 빠
    let rspMap = {
        '0': '5',
        '2': '0',
        '5': '2'
    }
    
    for(const ch of rsp) {
        answer += rspMap[ch]
    }
    return answer;
}